core/schema/internalsdk.graphqls

```diff
@@ -0,0 +1,138 @@
+extend type Query {
```

Comment:
The overall idea of this schema file is that the APIs are only supposed to be used internally by SDK implementations of Module support.

At a baseline, all of these APIs will return an error if called by a client besides the SDK.

We can also consider whether we want to hide or other-wise mark them as special in our codegen docs.